### ISSUE TITLE
Make the star button visible all the time

### ISSUE THREAD

**Issue by Razem** [maxneaga/open_dash_cam_android#30]
It would be nice to have an option (possibly using settings) to have the start button visible on the screen all the time. Preferably above the record button or next to it.

As of now, if you want to star a recording:

1. you have to click the record button, then the star button and then the record button again
2. or you need all the options visible all the time (meaning you click the record button one time and let it be)

The problem with the first option is that it takes really long. You have to make 3 clicks.

The problem with the second option is that it takes a lot of space on the screen. For example the current speed in Google Maps is blocked by a button.
